app/src/main/res/drawable/ic_merge_icon_24_disabled.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="36dp"
+  android:height="36dp"
+  android:viewportWidth="24"
+  android:viewportHeight="24">
+  <path
+    android:fillColor="#999999"
```

Comment:
Can we use an overlay color with the icon instead of duplicating the icon?

Reply:
tint color you mean right ?

Reply:
+1

Reply:
done